lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -79,14 +82,32 @@ def mulled_tags_for(namespace, image, tag_prefix=None):
 
 def split_tag(tag):
     """Split mulled image name into conda version and conda build."""
-    version = tag.split('--', 1)[0]
-    build = tag.split('--', 1)[1]
-    return version, build
+    return tag.split('--', 1)
```

Comment:
rsplit?

Reply:
don't want to change this, this is used in other places

Reply:
although that's probably safer, will do it